### ISSUE TITLE
two computer distributed Train model

### ISSUE THREAD

**Issue by feifaxiaoming** [tensorflow/models#4833]
there i have two machine for train model 
one is ubuntu  one is win10
i set as below:
 cluster = tf.train.ClusterSpec({
        "worker": [
            "xxxxx:2222",
            "xxxxxxx:3333",
        ],
    })
    job_name = 'worker'
    server = tf.train.Server(cluster, job_name=job_name, task_index=0)

i don't set ps job,
when i running my worker ,i found is slow step then on computer. 
my model is inception-v4.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by feifaxiaoming** [comment]
Have I written custom code:no
OS Platform and Distribution: master is ubuntu    other is win10 ;  there all worker  ,i don't set ps job.
TensorFlow installed from pip3 install tensorflow-gpu
TensorFlow version: ubuntu :1.3.0   win10:1.4.0
Bazel version:N/A
CUDA/cuDNN version :  8.0/6.0
GPU model and memory :GeForce GTX 1080  8GB
Exact command to reproduce:
model: inception-V4  slim

Exact command to reproduce:
cluster = tf.train.ClusterSpec({
"worker": [
"xxxxx:2222",
"xxxxxxx:3333",
],
})
job_name = 'worker'
server = tf.train.Server(cluster, job_name=job_name, task_index=0)

why i set two computer woker cluster , but is slower then one computer.
i train these model in one computer( INFO:tensorflow:global step 4898: loss = 0.7507 (0.370 sec/step))
i train two computer use cluster is  (0.891 sec/step)

it slower.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
